src/raftstore/store/msg.rs

```diff
@@ -124,6 +126,24 @@ pub enum Tick {
     CleanupImportSST,
 }
 
+impl Tick {
+    pub fn handle_with_tag<F, G>(self, f: F) -> G
```

Comment:
Why not `pub fn tag(&self) -> &'static str`?

Reply:
The rust-clippy will make it failed and force me to use `pub fn tag(self) -> &'static str`. 
If want to use the ref of a mutable static obj directly, you need unsafe code. However, when around with `RefCell`, it's impossible to use the ref outside.